### PR TITLE
check palindrome in a string in Java

### PR DESCRIPTION
**Pull request by raksha009**
Issue no #1718

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added a new code
- 

#### Languages Used:
- Java
- 

#### Files Added:
- palindrome_string.java
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Palindrome/palindrome_string.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+//Java program to check if a string is a Palindrome
+
+import java.util.*;
+
+public class palindrome_string {
+	
+	// Reverse a given string
+	public static String palindrome(String string_entered) {
+		String str_reversed  = "";
```
Comment (raghav-dalmia) [inline Palindrome/palindrome_string.java]: use 4 space indentation

File: Palindrome/palindrome_string.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			System.out.println(" String is a PALINDROME ");
+		}
+		else {
+			System.out.println(" String is not a PALINDROME ");
+		}
+	}
+}
+
+/*
+Test cases
```
Comment (raghav-dalmia) [inline Palindrome/palindrome_string.java]: mention input and output here

Comment (raksha009) [reply Palindrome/palindrome_string.java]: @raghav-dalmia  I have made the changes. plz review

File: Palindrome/palindrome_string.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+//Java program to check if a string is a Palindrome
+
+import java.util.*;
+
+public class palindrome_string {
+	
+    // Reverse a given string
+    public static String palindrome(String string_entered) {
```
Comment (raghav-dalmia) [inline Palindrome/palindrome_string.java]: change the function name to `reverse_string()`

**Comment by raghav-dalmia** [comment]
Also, squash your commits.

**Comment by raghav-dalmia** [comment]
your PR has 2 commits, squash them. I will merge it after that.

**Comment by raksha009** [comment]
@raghav-dalmia i have already sqaushed commits on my terminal. I have no idea why there are 2 commits shown here. Plz help how to solve this issue

**Comment by raksha009** [comment]
done @raghav-dalmia plz merge this PR. Thanks

**Comment by raksha009** [comment]
@raghav-dalmia plz have a look

**Pull request merged**